modules/core/02-client/client/cli/query.go

```diff
@@ -164,6 +164,48 @@ func GetCmdQueryConsensusStates() *cobra.Command {
 	return cmd
 }
 
+// GetCmdQueryConsensusStateHeights defines the command to query the height of every consensus states associated with a
+// given client state.
+func GetCmdQueryConsensusStateHeights() *cobra.Command {
+	cmd := &cobra.Command{
+		Use:     "consensus-state-heights [client-id]",
+		Short:   "Query the height of every consensus states of a client.",
+		Long:    "Query the height of every consensus states from a given client state.",
```

Comment:
```suggestion
		Long:    "Query the heights of all consensus states associated with the provided client ID.",
```

Reply:
thanks, my english is not as good as I thought